sdk/src/main/java/io/radar/sdk/model/RadarAddress.kt

```diff
@@ -130,21 +130,21 @@ class RadarAddress(
             }
 
             val coordinate = RadarCoordinate(obj.optDouble(FIELD_LATITUDE), obj.optDouble(FIELD_LONGITUDE))
-            val formattedAddress = obj.optString(FIELD_FORMATTED_ADDRESS)
-            val country = obj.optString(FIELD_COUNTRY)
-            val countryCode = obj.optString(FIELD_COUNTRY_CODE)
-            val countryFlag = obj.optString(FIELD_COUNTRY_FLAG)
-            val state = obj.optString(FIELD_STATE)
-            val stateCode = obj.optString(FIELD_STATE_CODE)
+            val formattedAddress = obj.optString(FIELD_FORMATTED_ADDRESS, null)
+            val country = obj.optString(FIELD_COUNTRY, null)
+            val countryCode = obj.optString(FIELD_COUNTRY_CODE, null)
+            val countryFlag = obj.optString(FIELD_COUNTRY_FLAG, null)
+            val state = obj.optString(FIELD_STATE, null)
+            val stateCode = obj.optString(FIELD_STATE_CODE, null)
             val postalCode = obj.optString(FIELD_POSTAL_CODE)
-            val city = obj.optString(FIELD_CITY)
-            val borough = obj.optString(FIELD_BOROUGH)
-            val county = obj.optString(FIELD_COUNTY)
-            val neighborhood = obj.optString(FIELD_NEIGHBORHOOD)
-            val street = obj.optString(FIELD_STREET)
-            val number = obj.optString(FIELD_NUMBER)
-            val addressLabel = obj.optString(FIELD_ADDRESS_LABEL)
-            val placeLabel = obj.optString(FIELD_PLACE_LABEL)
+            val city = obj.optString(FIELD_CITY, null)
+            val borough = obj.optString(FIELD_BOROUGH, null)
+            val county = obj.optString(FIELD_COUNTY, null)
+            val neighborhood = obj.optString(FIELD_NEIGHBORHOOD, null)
+            val street = obj.optString(FIELD_STREET, null)
```

Comment:
Testing with RN showed that we need the `null` fallbacks for all string params or `optString` will set an empty string `""` instead of `null`

Reply:
interesting, both are falsy values, so theoretically RN could do something like `if (address.neighborhood)` - but I guess the whole point is to move any serialization logic out of there..